hangman/README.md

```diff
@@ -1,3 +1,5 @@
 8k students only:
 
 Describe how to run the multinode app.
+
+I thought about making the Hangman restart strategy as :permanent (always restart), but like the Dictionary app, thought it better to use :transient and resart only when the application terminates abnormally
```

Comment:
:transient is probably the correct choice here, because the game can end, and the hangman process could potentially be changed to exit at some point in the future